src/components/timecard/edit-shift-hours/EditShiftHours.spec.jsx

```diff
@@ -530,6 +533,104 @@ describe('EditShiftHours', () => {
     expect(message).toBeTruthy();
   });
 
+  it('should show start and end dates when check box', async () => {
```

Comment:
Could we put the checkbox tests into their own describe block?

Reply:
Moved to their own section